frontend/src/metabase/qb/components/actions/XRayCard.jsx

```diff
@@ -8,7 +8,10 @@ import type {
 import { t } from "c-3po";
 import { utf8_to_b64url } from "metabase/lib/card";
 
-export default ({ question }: ClickActionProps): ClickAction[] => {
+export default ({ question, settings }: ClickActionProps): ClickAction[] => {
```

Comment:
I don't think `settings` is included in `ClickActionProps`?

Probably easiest to just do `MetabaseSettings.get("enable_xrays")`